README.md

```diff
@@ -195,6 +195,16 @@ languages([
 ])
 ```
 
+If you want to use a specific locale, you can also use 
```

Comment:
Could you append this directly to the already existing `languages` section of the README? https://github.com/KrauseFx/snapshot#languages